packages/toolkit/src/view/hub/hub-view/card-pipeline-hub/Head.tsx

```diff
@@ -39,24 +39,37 @@ export const Head = ({
     return null;
   }, [pipeline]);
 
+  // Function to transform the ID into a more readable format
+  const transformID = (id: string) => {
+    return id
+      .split(/[-_]/) // Split the ID by dash and underscore
+      .map((part) => part.charAt(0).toUpperCase() + part.slice(1)) // Capitalize each part
+      .join(" "); // Join the parts with a space
+  };
+
   const displayName = React.useMemo(() => {
     const owner = pipeline.owner;
 
     if ("user" in owner) {
-      // Owner is of type user
       const userOwner = owner as UserOwner;
       const userID = userOwner.user.name.split("/")[1]; // Get the part after "/"
```

Comment:
@thewbuk you can also do something like this
```suggestion
      const userID = userOwner.user.id
```